Cargo.toml

```diff
@@ -1,5 +1,6 @@
 [workspace]
 members = ["systeroid-core", "systeroid-tui", "systeroid"]
+resolver = "2"
```

Comment:
CI is failing because of this:

```
2025-09-01 16:10:55 [ERROR] failed to fetch crates: error: failed to parse lock file at: /github/workspace/Cargo.lock

Caused by:
  lock file version `4` was found, but this version of Cargo does not understand this lock file, perhaps Cargo needs to be updated?

2025-09-01 16:10:55 [ERROR] `cargo metadata` exited with an error: error: failed to parse lock file at: /github/workspace/Cargo.lock

Caused by:
  lock file version `4` was found, but this version of Cargo does not understand this lock file, perhaps Cargo needs to be updated?
```

Reply:
Since this is my first contribution in this project the workflow had to be approved. Thus I never saw these errors...